### DISCUSSION TITLE
Hero12 shut down delay

### DISCUSSION THREAD

**Discussion by mycosd** [gopro/labs#800] [Q&A]
Hi,

My naked Hero12 takes about 2min to shut down with the recent Labs firmware, is this normal behaviour? I've got Labs on another Hero12 (not naked yet), and it shuts down in less than 5sec. 
After it was shut down once, the next shut downs go quicker again... til I remove power. 

Thanks

**Comment by dnewman-gpsw** [comment]
Factory reset will likely help.  Scan !FRESET

**Reply by mycosd** [reply]
This didn't fix the shut down behaviour. 
Even before scanning my configuration afterwards, it still takes extremely long to shut down

**Reply by dnewman-gpsw** [reply]
Not normal, but if the camera still otehrwise works, don't worry about it.  I never shutdown my naked cameras.

**Reply by mycosd** [reply]
It works (kind of), it also has some other quirks related to this. It can't connect to the app on first start up. Only after the long shut down, when I start up again I can connect it to the App. IMHO I think there is a service (eg. Bluetooth) not starting up correctly, and when it should shut down, it wants to stop the service that's not working.
I'm a bit hesitant with removing power, because of Quick Capture you'll never really know if it's still writing to the card.

**Reply by dnewman-gpsw** [reply]
I never use QuikCapture, using audio or IMU triggers to automatically start and stop captures for naked cameras.  Even if recording is still happening, the file is repaired automatically on the next boot.  It does sound like the camera is little unwell, did you update the firmware?

**Reply by mycosd** [reply]
The cam is about 1 month old. I updated it to the recent GoPro firmware when it arrived, just to check if it is working. Then I installed Labs and it was still working fine. Then I de-cased it, and now it's wonky, although I'm pretty sure that's just a software issue.

**Reply by dnewman-gpsw** [reply]
It is way more likely to be the hardware issue than the software, as a Factory Reset wipes any soft configurations.

**Reply by mycosd** [reply]
The reason I don't think it's a hardware issue, is because everything works. Just not on the first boot.
Also I just discovered that when I use `!MWAKE=2`, the issue is totally gone, and I can power down in a few seconds and the app connection issue is gone as well.

**Reply by dnewman-gpsw** [reply]
Well good, as all naked cameras should be using !MWAKE=2
